authtools/views.py

```diff
@@ -61,8 +64,8 @@ class WithNextUrlMixin(object):
     success_url = None
 
     def get_next_url(self):
-        if self.redirect_field_name in self.request.REQUEST:
-            redirect_to = self.request.REQUEST[self.redirect_field_name]
+        if self.redirect_field_name in self.request.GET:
+            redirect_to = self.request.GET[self.redirect_field_name]
```

Comment:
There's no real technical reason except making the code cleaner/smaller.

But I think that it's more semantic. And you can POST to `/login/?next=foo`, so why would you put it in the POST.